docs/install/installing-istio.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Installing Istio for Knative"
+weight: 15
+type: "docs"
+---
+
+This guide walks you through manually installing and customizing Istio for use
+with Knative.
+
+If your cloud platform offers a managed Istio installation, we recommend
+installing Istio that way, unless you need the ability to customize your
+installation. If your cloud platform offers a managed Istio installation,
+the [install guide](./) for your specific platform will have those instructions.
```

Comment:
@RichieEscarez  -- so these links 404 when browsing on GitHub; but I think they'd work on the site. I forget, should I be using the GitHub-appropriate URLs, and then the script you made a while back makes the links appropriate for the website at build time?

Reply:
Use GitHub-appropriate links (sorry ive not gotten back to finishing the authors guide). So `[install guide](./README.md)` is what you should use and the site build script fixes it at build time for knative.dev.

Reply:
Cool, that's what I thought. Thanks!